src/audit-log-portal/src/components/SearchResults/MessagesSearchResults.tsx

```diff
@@ -0,0 +1,51 @@
+import type MessageSearchModel from "types/MessageSearchModel"
+import useGetMessages from "utils/useGetMessages"
+import Messages from "../Messages"
+import convertObjectToURLSearchParams from "utils/convertObjectToURLSearchParams"
+import combineUrlAndQueryString from "utils/combineUrlAndQueryString"
+import InfiniteScroll from "react-infinite-scroll-component"
+import If from "../If"
+import Loading from "../Loading"
+import { Fragment } from "react"
+import Error from "components/Error"
+import type { Props } from "./SearchResults"
+
+const resolveMessagesApiUrl = (searchModel: MessageSearchModel, lastMessageId?: string): string => {
+  // map MessageSearchModel fields to messages endpoint query parameters
+  const searchParams = {
+    status: searchModel.status ?? "",
+    externalCorrelationId: searchModel.searchId ?? ""
+  }
+
+  const params = convertObjectToURLSearchParams(searchParams)
+  if (lastMessageId) {
+    params.append("lastMessageId", lastMessageId)
+  }
+  return combineUrlAndQueryString(`/audit-logging/api/messages`, params.toString())
+}
+
+const MessagesSearchResults = ({ searchModel }: Props) => {
+  const { messages, error, loadMore, isLoadingInitialData, isLoadingMore, isReachingEnd, reload } = useGetMessages(
+    (_, previousMessages) => {
+      //@ts-ignore
```

Comment:
Great work on this PR! I'm curious, what exactly are we ignoring here? I wonder if we could find a solution rather than use `//@ts-ignore`

Reply:
thanks for the review! I'm not sure what it's ignoring, it was carried over from https://github.com/ministryofjustice/bichard7-next-audit-logging/blob/381da38122c262d9db48416b07af08c640fdeb49/src/audit-log-portal/src/pages/index.tsx#L28 but it can be removed now without setting off the linter or causing any problems 🤷‍♀️